src/pages/registration/DescriptionPanel.tsx

```diff
@@ -81,16 +73,24 @@ export const DescriptionPanel = () => {
                   label={t('registration.description.alternative_title')}
                 />
               ) : null}
+              {field.value || field.value === '' ? null : (
+                <Button
+                  startIcon={<AddCircleOutlineIcon />}
+                  disabled={field.value !== undefined || !values.entityDescription?.mainTitle}
```

Comment:
dette sjekkes på linje 76 i stedet nå
```suggestion
                  disabled={!values.entityDescription?.mainTitle}
```